user/src/com/google/gwt/user/server/rpc/impl/ServerSerializationStreamReader.java

```diff
@@ -367,7 +366,7 @@ Object read(ServerSerializationStreamReader stream, BoundedList<Object> instance
 
   private final ClassLoader classLoader;
 
-  private SerializationPolicy serializationPolicy = RPC.getDefaultSerializationPolicy();
```

Comment:
This inlining should be landed separately, like the other work done in https://github.com/gwtproject/gwt/pull/9826